### PR TITLE
Clarify what TFM stands for

### PR DESCRIPTION
**Pull request by davidfowl**
https://github.com/dotnet/corefx/issues/4367#issuecomment-175606321

/cc @Petermarcu @richlander @ericstj

### CODE REVIEW

**Comment by ericstj** [comment]
Conflicts with https://github.com/dotnet/corefx/pull/5669.

**Comment by davidfowl** [comment]
:\

File: Documentation/project-docs/standard-platform.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The .NET Platform Standard version represents binary portability across platform
 - **.NET Platform Standard** - a specific versioned set of reference assemblies that all .NET Platforms must support as defined in the [CoreFX repo](https://github.com/dotnet/corefx).
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, .NET Framework 4.6, Windows Phone 8.1, MonoTouch, UWP, etc.
```
Comment (ericstj) [inline Documentation/project-docs/standard-platform.md]: I don't think I agree with "Platform" definition here.  Elsewhere platform is used to represent 'OS' and framework represents this concept.  That's why the 'f' in TFM is for framework and it distinguishes it from TPM where 'p' is platform.

I'm also not sure why we are adding 'platform' into .NET Standard above.

Comment (davidfowl) [inline Documentation/project-docs/standard-platform.md]: The problem is that .net standard is a different kind of TFM the way it's laid out here. The intent was to call that out in the doc.

**Pull request closed without merging**